charts/operator/README.md

```diff
@@ -116,15 +122,24 @@ matchers:
             value: "<YOUR TOKEN>"
 ``` 
 
-## Check deployment status
+## Important log lines
 
-Get all deployment logs :
+How to see the operator logs ?
```

Comment:
I think ":" is more suitable here than "?", or neither.

Also, why is there a space before the "?"